provisioner/ansible/test-fixtures/long-debug-message.yml

```diff
@@ -0,0 +1,8 @@
+- name: Stub for Packer testing long Ansible messages
+  hosts: localhost
+  connection: local
+
+  tasks:
+    - name: Very long Ansible output (>65535 chars) (Issue https://github.com/mitchellh/packer/issues/3268)
+      debug:
+        msg: "{{ lipsum(n=300, html=false) }}"
```

Comment:
Is lipsum a built-in ansible feature or do we need to make sure this is installed somehow?